src/app/shared/actions/volume-actions/volume-resize.container.ts

```diff
@@ -29,6 +30,10 @@ import { VolumeType } from '../../models';
 export class VolumeResizeContainerComponent implements OnInit {
   readonly offerings$ = this.store.pipe(select(fromDiskOfferings.getAvailableOfferings));
   readonly account$ = this.store.pipe(select(fromAuth.getUserAccount));
+  readonly availableStorage$ = this.account$.pipe(
```

Comment:
Can't you make an `availableStorage` selector? You use this construct at least in two places.